core/modules/views_ui/views_ui.admin.inc

```diff
@@ -1059,7 +1059,8 @@ function views_ui_edit_form($form, &$form_state, $view, $display_id = NULL) {
   $form['#prefix'] = $form['#prefix'] . '<div class="views-edit-view views-admin clearfix">';
   $form['#suffix'] = '</div>' . $form['#suffix'];
 
-  $form['#attributes']['class'] = array('form-edit');
+  $form['#attributes']['class'][] = 'form-edit';
+  $form['#attributes']['class'][] = 'sticky-enabled';
```

Comment:
I'm fine with this, but why not use the existing array to stick the new class in?:

```suggestion
  $form['#attributes']['class'] = array('form-edit', 'sticky-enabled');
```